app/retail/templates/home/nav.html

```diff
@@ -16,8 +16,8 @@
 
 {% endcomment %}
 {% load i18n static matches cookielaw_tags %}
-<nav class="navbar navbar-expand-md navbar-light bg-white px-3 px-md-1">
-  <a class="navbar-brand py-0 mr-0" href="/landing">
+<nav class="navbar navbar-expand-md navbar-light bg-white p-0 px-md-1">
+  <a class="navbar-brand sticky-top py-0 mr-0 bg-white my-md-auto" href="/">
```

Comment:
can you use `{% url 'index' %}` ?

Reply:
Resolved in 7cd0d2e3bf96cc58f09dafa0833650d0731f2c7b